sphinx/source/docs/user_guide/bokehjs.rst

```diff
@@ -280,6 +280,78 @@ and hover policy. Here is an example of a ``bar`` chart and the plot it generate
 
     plt.show(plt.gridplot([[p1, p2], [p3, p4]], {plot_width:350, plot_height:350}));
 
+.. _userguide_bokehjs_differences:
+
+Usage : Comparison to Python
```

Comment:
I believe there is an extra whitespace after "Usage". Also: I feel like "comparison" is not the optimal word here - I'd prefer something like differences. How about:

> Differences to Python library